### ISSUE TITLE
Ctrl-C not passed correctly to Windows executable.

### ISSUE THREAD

**Issue by alexander-koerschgen** [git-for-windows/git#1317]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.14.1.windows.1
built from commit: 82d9b3f3b2407b52251620597d4b14933685459d
sizeof-long: 4
machine: x86

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 6.1.7601]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
$ cat /etc/install-options.txt

Path Option: Cmd
SSH Option: OpenSSH
CURL Option: OpenSSL
CRLF Option: LFOnly
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

Windows Installation of TeXLive present.
$ pdflatex -v
pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017/W32TeX)
[...]

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Git bash (as shipped with Git for Windows)

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
In a directory containing a LaTeX document,
$ latexmk -pvc
Then hit Ctrl-C to stop PVC ("preview continously") mode.
```
 - What did you expect to occur after running these commands?

The message "Latexmk: User typed ctrl/C or ctrl/break.  I'll finish." should appear on the terminal, indicating that latexmk was able to capture the Ctrl-C. Afterwards, there should be no processes related to latexmk left. This is the intended behavior that occurs on Linux and on Windows using a Cmd, Cygwin, or MinGW shell.

 - What actually happened instead?

There is no message on the terminal which returns to the usual prompt. However, a perl.exe process, which is related to latexmk, remains running and watches for changes in the LaTeX project.

 - Investigation so far:

Latexmk.exe calls .../texlive/2017/tlpkg/tlperl/bin/perl.exe to run a script called latexmk.pl. This version of perl reports
"This is perl 5, version 24, subversion 1 (v5.24.1) built for MSWin32-x86-multi-thread"
If one uses the perl executable that comes with Git bash (/usr/bin/perl) to run latexmk.pl, the problem does not occur.

In conclusion, there must be a problem in Git bash with passing Ctrl-C to the Windows executable .../texlive/2017/tlpkg/tlperl/bin/perl.exe. I'd like to stress that this does not occur in a MinGW shell, which seems to be similar to Git bash, although I do not understand the details.

**Comment by dscho** [comment]
There are several still open tickets about this issue.

**Comment by michaelblyons** [comment]
Alexander, have you tried this since 2.16.0(2)? I had similar issues until that release.

**Comment by alexander-koerschgen** [comment]
Thank you for bringing this up, Michael. I just ran a few tests on version 2.16.1 and the issue seems to be fixed!
